components/CardNews/src/index.scss

```diff
@@ -0,0 +1,90 @@
+.denhaag-card-news {
+  text-decoration: var(--denhaag-card-news-text-decoration);
+}
+
+.denhaag-card-news__figure {
+  border: var(--denhaag-card-news-figure-border);
+  padding-block-start: var(--denhaag-card-news-figure-padding);
+  padding-block-end: var(--denhaag-card-news-figure-padding);
+  padding-inline-start: var(--denhaag-card-news-figure-padding);
+  padding-inline-end: var(--denhaag-card-news-figure-padding);
+  margin-inline-start: var(--denhaag-card-news-figure-margin);
+  margin-inline-end: var(--denhaag-card-news-figure-margin);
+  margin-block-start: var(--denhaag-card-news-figure-margin);
+  margin-block-end: var(--denhaag-card-news-figure-margin);
+  min-width: var(--denhaag-card-news-figure-min-width);
+  width: var(--denhaag-card-news-figure-width);
```

Comment:
Shouldn't the width be defined by the grid where it's in? Otherwise you'll get issues with responsiveness